scripts/install

```diff
@@ -176,7 +175,13 @@ download_binary() {
 		GVM_ARCH="386"
 	fi
 
-	GO_BINARY_FILE=${VERSION}.${GVM_OS}-${GVM_ARCH}${GVM_OS_VERSION}.tar.gz
+	extract_version $VERSION
+	compare_version $SEMVER "1.4.3"
```

Comment:
Agreed, it felt a little weird for me too. I suppose I can make "extract_version" `echo` calculated version and assign that to `SEMVER`. Will have a go at this later